lib/plug/ssl.ex

```diff
@@ -102,11 +106,15 @@ defmodule Plug.SSL do
 
   defp put_hsts_header(conn, _), do: conn
 
-  defp redirect_to_https(%Conn{host: host} = conn, custom_host) do
+  defp redirect_to_https(%Conn{host: host} = conn, custom_host, log_level) do
     status = if conn.method in ~w(HEAD GET), do: 301, else: 307
 
     location = "https://" <> host(custom_host, host) <> conn.request_path <> qs(conn.query_string)
 
+    Logger.log(log_level, fn ->
```

Comment:
I think we should say something like "Plug.SSL is redirecting GET /PATH to https://HOST with status STATUS"